packages/node_modules/nav-frontend-alertstriper/src/advarsel-alertstripe.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="react" />
```

Comment:
Fint hvis du fjerner alle disse definisjonsfilene. De blir generert automatisk ved bygg, så de trenger vi ikke :) Forøvrig litt rart at de ligger under `src` hos deg. Bruker du `npm run build` i rot-katalogen for å bygge `.tsx`-filene?

Reply:
Har brukt `npm run build` ja, før jeg ble fortalt om `npm run gulp build`. `start-guideline-app` fungerte dårlig for meg.

Reply:
Hva var det som ikke fungerte med `start-guideline-app`?